samples/net/openthread/coap_client/README.rst

```diff
@@ -0,0 +1,68 @@
+.. _coap_client_sample:
+
+openthread: coap_client
+##################
+
+The openthread coap_client sample demonstrates interactions between nodes performing different Thread roles with the use of OpenThread and CoAP protocol.
+
+Overview
+********
+
+The client node can control the state of BSP_LED_3 on server nodes. It can turn on the LED either on every server node in the network with a multicast message, or on a single specific server node that is paired with the client node.
+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * |nRF52840DK|
+  * |nRF52833DK|
+
+User interface
```

Comment:
I would add description of BUTTON 4, LED 3 and LED 4 here. Then just mention LIGHT_LED and PROVISIONING_LED in the testing section.